### DISCUSSION TITLE
A huge problem with the connected Github account on the Vercel!!!

### DISCUSSION THREAD

**Discussion by Samir-Saber** [vercel/community#4874] [Account Issue] [answered]
Hi All.
I need your help, please. it is a critical problem.

our website is hosted on Vercel and was deployed by the developer with his own Github account.

Now the developer is not available and we don't have access to his account.

Is it fine to disconnect his Github account? what will happen if we diconnect it? will the website go down?

If fine to disconnect his account from the Vercel! will we able to connect it to another Github account for content management purposes?

Please help me.

Thank you in advance.

**Comment by amyegan** [comment, marked as answer]
Hi @ArscoIT. Existing deployments do not get deleted when you disconnect a GitHub account. 

If you have a copy of the repo on another account, you can connect to that one instead. If you don't have a copy of the repo, then you won't be able to build new deployments for content updates.

**Reply by Samir-Saber** [reply]
@amyegan Thank you for your reply, 

We don't have another copy on another account, we have a local copy, can we use it? like upload it to our Github account then connect it to the Vercel server?

How can we get benefits of the local copy?

Your attention to my case is appreciated.

**Reply by amyegan** [reply]
No worries! You could push the local copy to a new GitHub repo and connect it that way. Or you can use [Vercel CLI](https://vercel.com/docs/cli) to manually deploy from a local copy. 

https://vercel.com/docs/cli/deploying-from-cli

**Reply by Samir-Saber** [reply]
Thank you very much Amyegan for the informations.

I appreciated your help.

Best Regards,
